source/content/guides/integrated-composer/03-private-repo-package.md

```diff
@@ -18,15 +18,23 @@ integration: [--]
 
 This section provides information on how to add a package from a private repository using Integrated Composer.
 
+<Alert title="Note" type="info">
+
+Pantheons Secrets Manager is currently in Early Access (EA). You can use the [Terminus plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin) to manage your secrets for private repositories in Integrated Composer builds.
```

Comment:
This note should also mention that users who wish to use the Customer Secrets service should follow the instructions in the [README of the Terminus plugin](https://github.com/pantheon-systems/terminus-secrets-manager-plugin#use-secrets-with-integrated-composer) **instead of** the documentation here.